src/java/org/broadinstitute/dropseqrna/utils/SplitBamByCell.java

```diff
@@ -70,9 +73,12 @@
     @Argument(doc="The tag to examine in order to partition reads.")
     public String SPLIT_TAG="XC";
 
-    @Argument(doc="Number of output files to create")
+    @Argument(optional=true, doc="Number of output files to create")
     public Integer NUM_OUTPUTS;
 
+    @Argument(optional=true, doc="Approximate size of split BAMs to be created. This can be a human-readable number like 500M or 2G. Default: 2G")
```

Comment:
Note that the message saying that there is a default value is not accurate.  There is no default.